pages/index.tsx

```diff
@@ -147,68 +219,134 @@ export default function Home() {
+              </div>
+              <p>
+                The result of the database query is cached at the Accelerate caching node in&nbsp;
+                <span className={styles.badgeYellow}>
+                  <img src="/locationDot.svg" color={defaultTheme.colors.orange[400]} width="8px" height="15px" />
+                  {history?.[0]?.location ?? ""}
+                </span>&nbsp;
+                and retrieved from there:
+              </p>
+              <ul>
+                <li>✨ Reduced latency</li>
+                <li>🚀 Increased Query Capacity</li>
+                <li>🌟 Optimal Resource Utilization</li>
+              </ul>
+              <div className={`${styles.expandBar} ${styles.with} ${showWith && styles.active}`} onClick={() => toggleWith(!showWith)}>
+                Expand to view Prisma Client query
+              </div>
+              <pre className={`${styles.code} ${!showWith && styles.hide}`}>
+                <code>
+                  {CODE_CACHE}
+                </code>
+              </pre>
+            </div>
+          </div>
+          <div className={styles.withoutAccelerate}>
+            <h4><img src="/clock.svg" /> Without Accelerate Cache</h4>
+            <div className={styles.illustrationSection}>
+            <Animation autoplay={true} name="without-accelerate" className={styles.animation} fit={reactCanvas.Fit.FitWidth} />
+            </div>
+            <div className={styles.cardInfo}>
+              <div className={styles.numbers}>
+                <h4>{num.format((1_000 / withoutCacheLatency) * 60)}</h4>
+                <span>queries per minute</span>
+              </div>
+              <div className={styles.numbers}>
+                <h4>{ms.format(withoutCacheLatency)}</h4>
+                <span>latency</span>
+              </div>
+              <p>
+              The database query and its response need to travel to the database in&nbsp;
+                <span className={styles.badgeYellow}>
+                  <img src="/locationDot.svg" color={defaultTheme.colors.orange[400]} width="8px" height="15px" />
+                  {history?.[0]?.location ?? ""}
+                </span>&nbsp;
+                and retrieved from there:
+              </p>
+              <ul>
+                <li>🐢 High latency</li>
+                <li>🪫 Low Query Capacity</li>
+                <li>🚧 Poor Resource Utilization</li>
+              </ul>
+              <div className={`${styles.expandBar} ${styles.without} ${showWithout && styles.active}`} onClick={() => toggleWithout(!showWithout)}>
+                Expand to view Prisma Client query
+              </div>
+              <pre className={`${styles.code} ${!showWithout && styles.hide}`}>
+                <code>
+                  {CODE_NO_CACHE}
+                </code>
+              </pre>
+            </div>
+          </div>
+        </div>
+        <div className={styles.results}>
+          <h4>Speed test history</h4>
```

Comment:
the column alignment of both tables ("Speed test history" and "Database instance used") is a bit glitchy right now:

<img width="874" alt="image" src="https://github.com/prisma/accelerate-speed-test/assets/4058327/b32350a5-64e6-4314-bc21-ca7cbb65ee7c">